.travis.yml

```diff
@@ -1,6 +1,22 @@
-language: node_js
-node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-  - "4.2.4"
+language: c++
+env:
+  - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
+  - NODE_VERSION="0.12"
+  - NODE_VERSION="4.2.4"
```

Comment:
@nfischer: I know, but change this from `NODE_VERSION="4.2.4"` to just `NODE_VERSION="4"`. That way if they do a patch release it will automatically work.